### DISCUSSION TITLE
Question: on_motion_detected

### DISCUSSION THREAD

**Discussion by myxxmikeyxx** [Motion-Project/motion#1438] [Q&A]
Are you able to make it run two files by using the &
Example `on_motion_detected /usr/bin/python /home/pi/mute.py & /bin/bash /home/pi/play.sh`
I just can not get it to work and didn't know if I needed to format it different or if its not supported

**Comment by cy2201** [comment]
Do you have a shebang in your play.sh? 
You could create a third file and call these two scripts inside this. Then just call this created file in motion.
